x-pack/platform/plugins/shared/screenshotting/server/formats/pdf/pdf_maker/pdfmaker.ts

```diff
@@ -254,7 +254,16 @@ export class PdfMaker {
         const generatePdfRequest: GeneratePdfRequest = {
           data: this.getGeneratePdfRequestData(),
         };
-        myPort.postMessage(generatePdfRequest, this.transferList);
+        try {
+          myPort.postMessage(generatePdfRequest, this.transferList);
+        } catch (e) {
+          // See https://github.com/nodejs/node/issues/55593
+          if (e.name === 'DataCloneError') {
+            myPort.postMessage(generatePdfRequest);
```

Comment:
Probably not the fix we're looking for.

https://github.com/elastic/kibana/pull/205983#issuecomment-2581529383

Reply:
I took a different approach to tackle the issue [here](https://github.com/jbudz/kibana/pull/3), so the referenced buffers get transferred as intended.